src/app/routes/onDemandTV/getInitialData/index.js

```diff
@@ -28,6 +28,23 @@ const getDurationISO8601 = path([
 const getThumbnailImageUrl = json =>
   getPlaceholderImageUrl(path(['promo', 'media', 'imageUrl'], json));
 const getPromoBrandTitle = path(['promo', 'brand', 'title']);
+const getImageUrl = path(['content', 'blocks', 0, 'imageUrl']);
```

Comment:
It looks like the image URL contains a `$recipe` placeholder - does this need to be replaced here?

Reply:
We replace it in the VideoPlayer component:  (https://github.com/bbc/simorgh/pull/6710/files/41a41beb5a772eeb9c2ce3007d1cf8ee5dc1f20d#diff-ca2e92834cd214b2fba577c29704dde8R65)

using this: https://github.com/bbc/simorgh/blob/latest/src/app/routes/utils/getPlaceholderImageUrl/index.js#L5

Reply:
Thanks!